src/Model/Conversation.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * @file src/Model/Conversation
+ */
+namespace Friendica\Model;
+
+use Friendica\Database\DBM;
+use dba;
+
+require_once "include/dba.php";
+
+class Conversation
+{
+	/**
+	 * @brief Store the conversation data
+	 *
+	 * @param array $arr Item array with conversation data
+	 * @return array Item array with removed conversation data
+	 */
+	public static function insert($arr) {
```

Comment:
We should agree on a common naming scheme for Model classes, currently we use `create`, `add` and `insert` for the same operation.

Reply:
I would like to orientate after the SQL commands, mean: "insert", "update" and "delete". 

BTW: You forgot ```store``` (as seen in Model/Photo.php) :-)

Reply:
But `store` is a combination of `insert/update`, so not strictly the same as all the above!

Reply:
Okay. What would you suggest?

Reply:
https://friendica.mrpetovan.com/display/735a2029915a63dda2c8b7b601667320